src/structures/guild.ts

```diff
@@ -385,6 +385,42 @@ export class Guild extends Base {
 
     return result.pruned as number
   }
+
+  async prune(
+    days?: number,
+    computePruneCount?: false,
+    includeRoles?: Array<Role | string>
+  ): Promise<null>
+  async prune(
+    days?: number,
+    computePruneCount?: true,
+    includeRoles?: Array<Role | string>
+  ): Promise<number>
+  async prune(
+    days?: number,
+    computePruneCount?: undefined,
+    includeRoles?: Array<Role | string>
+  ): Promise<number>
+  async prune(
+    days?: number,
+    computePruneCount?: boolean,
+    includeRoles?: Array<Role | string>
+  ): Promise<number | null> {
```

Comment:
What...

Reply:
This is to show user different return types per `computePruneCount`. I didn't have better idea than that.